library/ajax/pharmacy_autocomplete/search.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * weno rx search.
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once('../../../interface/globals.php');
+
+
+if (isset($_GET['term'])) {
+    $return_arr = array();
+    $term    = filter_input(INPUT_GET, "term");
+    $city    = filter_input(INPUT_GET, "city");
+    $address = filter_input(INPUT_GET, "address");
+
+    try {
+        $sql = "SELECT id, Store_name, address_line_1, city, state FROM erx_pharmacies WHERE Store_name LIKE ? AND city LIKE ? ";
+        $sql .= " AND address_line_1 LIKE ? ";
+        $stm = array('%'.$term.'%','%'.$city.'%','%'.$address.'%');
+        $res = sqlstatement($sql, $stm);
+        while ($row = sqlFetchArray($res)) {
+            $return_arr[] =  $row['id'] . " - ".$row['Store_name'] . " ".$row['address_line_1']." ". $row['city'] ." " . $row['state'];
+        }
+    } catch (PDOException $e) {
```

Comment:
This seems odd since the database connector is not even PDO???

Reply:
It is a left over from the original code.

Reply:
then would completely remove this try/catch (note sqlStatement outputs errors already)
as an aside, does these even work since the S in sqlstatement is not capitalized??

Reply:
Will remove try catch blocks.